packages/cli/src/cmds/preview/constants.ts

```diff
@@ -120,8 +120,8 @@ const loopbackAddressIPv6 = "::1";
 const hosts = [allAddressIPv4, loopbackAddressIPv4, allAddressIPv6, loopbackAddressIPv6];
 
 export const frontendPorts: [number, string[]][] = [
-  [3000, hosts],
-  [5000, hosts],
+  [53000, hosts],
```

Comment:
Can we move both `53000` and `55000` to a common constant so that CLI/VSC/FX-Core can reuse it instead of hard-code in multiple places?

Reply:
Absolutely, see the PR linked by Qianhao below. Port checker will be a utility in core for the middle term plan.